src/control/control.cpp

```diff
@@ -102,7 +102,10 @@ QSharedPointer<ControlDoublePrivate> ControlDoublePrivate::getControl(
         bool bTrack,
         bool bPersist,
         double defaultValue) {
-    VERIFY_OR_DEBUG_ASSERT(key.isValid()) {
+    if (!key.isValid()) {
+        if (!flags.testFlag(ControlFlag::AllowInvalidKey)) {
+            DEBUG_ASSERT(key.isValid());
```

Comment:
```suggestion
            DEBUG_ASSERT(flags.testFlag(ControlFlag::AllowInvalidKey));
```

Reply:
Hmm, I understand how that makes the code a little clearer to read, but on the other hand, I think it makes the error message from a failed assertion harder to understand without looking at the code.

Reply:
If you move it below the warning everything should be clear.

Reply:
Sure, that works